pandora.py

```diff
@@ -272,6 +272,27 @@ def __init__(self):
             'musicId': music_id,
         })
 
+    async def get_explicit_content_filter(self) -> bool:
+        """
+        Get if the explicit content filter is enabled.
+        """
+        response = await self._send_message('v1/listener/updateSettings', {})
```

Comment:
Again I wonder if these should be the generic apis of just `update_settings` and let you pass any name you want?